libmscore/style.cpp

```diff
@@ -2381,8 +2431,12 @@ void MStyle::load(XmlReader& e)
                   set(Sid::ottavaHookAbove, y);
                   set(Sid::ottavaHookBelow, -y);
                   }
-            else if (tag == "Spatium")
-                  set(Sid::spatium, e.readDouble() * DPMM);
+            else if (tag == "spatium") { // 3.01+ lowercase
```

Comment:
I don't like the same names which are different by capitalisation. Better name is needed. "SpatiumNew", "UniversalSpatium"?